app/components/map/index.js

```diff
@@ -408,9 +451,13 @@ class Map extends Component {
   }
 
   render() {
-    const { hasCompass, lastPosition, compassFallback, selectedAlertCoordinates } = this.state;
+    const { hasCompass, lastPosition, compassFallback, selectedAlertsCoordinates, neighbours } = this.state;
     const { areaCoordinates, datasetSlug } = this.props;
-    const showCompassFallback = !hasCompass && lastPosition && selectedAlertCoordinates && compassFallback;
+    const showCompassFallback = !hasCompass && lastPosition && selectedAlertsCoordinates && compassFallback;
```

Comment:
This won't work now, because selectedAlertsCoordinates is now an array, we'll need to check for the length of it. I'll make the changes ✌️